clients/js/test/startChromaContainer.ts

```diff
@@ -0,0 +1,81 @@
+import path from "node:path";
+import { GenericContainer, Wait } from "testcontainers";
+
+const CHROMADB_PORT = 8000;
+
+const BUILD_CONTEXT_DIR = path.join(__dirname, "../../..");
+
+const BASIC_AUTH_FILE = path.join(__dirname, "test.htpasswd");
```

Comment:
maybe it'd be good to create this dynamically rather than checking it in? can't see what the password is by looking at `test.htpasswd`

Reply:
Good call! Test containers allows you to just copy strings to files in the image so we don't even need that htpasswd file at all.